ext/mbstring/tests/bug43994.phpt

```diff
@@ -49,7 +49,8 @@ With $regs arg:
 
 Warning: mb_ereg(): empty pattern in %s on line %d
 bool(false)
-NULL
+array(0) {
+}
```

Comment:
>  Ah I see, you want to match the ereg api.

No, I don't want to, but the current behavior of mb_ereg() matches that of ereg() (as far as I can tell), but not the documentation of mb_ereg(). Consider "$regs: Contains a substring of the matched string." That's simply nonsense; even "contains substrings of the matched string" would still be wrong, as $regs[0] is supposed to contain the full match.